cmd/podman/containers/run.go

```diff
@@ -77,6 +77,11 @@ func runFlags(cmd *cobra.Command) {
 	flags.StringVar(&runOpts.DetachKeys, detachKeysFlagName, containerConfig.DetachKeys(), "Override the key sequence for detaching a container. Format is a single character `[a-Z]` or a comma separated sequence of `ctrl-<value>`, where `<value>` is one of: `a-cf`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`")
 	_ = cmd.RegisterFlagCompletionFunc(detachKeysFlagName, common.AutocompleteDetachKeys)
 
+	gpuFlagName := "gpus"
+	flags.String(gpuFlagName, "", "This is a Docker specific option and is a NOOP")
```

Comment:
I wonder if it's worth printing a log message (Info level, probably) if this is actually set, notifying the user it has no effect

Reply:
Added info level log message if the flag is set; message follows description of flag.

Reply:
Hide it, no reason for it other then backwards compatibility with Docker CLI.

Reply:
@rhatdan Did you mean hide the flag from `--help`?

Reply:
Hide the command from --help